### ISSUE TITLE
Request for a new release of Osmosis package

### ISSUE THREAD

**Issue by munterfi** [openstreetmap/osmosis#128]
The current release of the osmosis package on GitHub was back in September 2020 wit version `0.48.3`, and there hasn't been any new release since then. As it has been over two years since the last release,  the current released version has outdated dependencies, which are already updated in the main branch of the package (e.g. `org.springframework:spring-core:5.1.0.RELEASE` -> `dependencyVersionSpring=5.3.20`).

Are any plans for a new release of the package? Thank you!

**Comment by brettch** [comment]
I've just released a new version from the main branch. I hope it works 🤞 

It's 0.49.1 on Maven Central.

**Comment by brettch** [comment]
Hmm, that's a good question. Is that a problem for many dependencies or just those specific dependencies? I don't _think_ I did anything to change it.

Perhaps I inadvertently changed how dependencies worked as part of updating to Gradle 8. I know the Maven publishing process broke and I had to figure that out at the last minute.

I don't have a lot of time to investigate unfortunately. I'm not writing any other Java software at the moment and not actively using Osmosis. It takes me quite a while to get familiar with the latest versions of tooling and get a release out. If you have any suggestions (or better yet a PR) I'd really appreciate it.
